packages/jest-circus/src/utils.ts

```diff
@@ -28,12 +28,12 @@ export const makeDescribe = (
   }
 
   return {
+    type: 'describeBlock', // eslint-disable-next-line sort-keys
```

Comment:
disabling next line works out quite nicely for this rule, you can associate it with the line that you want to take out of the flow, because the error would be on the next line :sweat_smile: